cherrypy/test/test_states.py

```diff
@@ -112,7 +112,7 @@ def test_0_NormalStateFlow(self):
 
         host = cherrypy.server.socket_host
         port = cherrypy.server.socket_port
-        self.assertRaises(IOError, portend._check_port, host, port, timeout=0.1)
+        portend.occupied(host, port, timeout=0.1)
```

Comment:
I realized that what the test was doing here was not to test the behavior of _check_port, but was simply asserting that the port was occupied by the server above. `portend.occupied` does exactly this.